Kinvey/Kinvey/Error.swift

```diff
@@ -89,6 +89,14 @@ public enum Error: Swift.Error, CustomStringConvertible, CustomDebugStringConver
         }
     }
     
+    public var errorDescription: String? {
+        return description
+    }
+    
+    public var failureReason: String? {
+        return description
```

Comment:
Is there much value having `errorDescription` and `failureReason` be the same thing?